routes/users.js

```diff
@@ -0,0 +1,84 @@
+import express from 'express';
+import bcrypt from 'bcryptjs';
+
+import { addUserValidationRules, validateData } from '../validate.js';
+import { User } from '../models';
+
+const router = express.Router();
+
+const getUsers = async (request, response) => {
+  try {
+    const users = await User.findAll();
+    response.status(200).send(users);
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const addUser = async (request, response) => {
+  const { userName, email, firstName, lastName, password } = request.body;
+
+  try {
+    const salt = await bcrypt.genSalt(10);
+    const hashPassword = await bcrypt.hash(password, salt);
+    await User.create({
+      firstName,
+      lastName,
+      email,
+      password: hashPassword,
+      userName,
+    });
+    response.status(201).json({
+      status: 'Success',
+      message: 'User added.',
+    });
+  } catch (error) {
+    console.log(error);
```

Comment:
What will be the response in case of an error. You should either handle the error and send a response with a message.

Reply:
Please make sure you handle this everywhere.